index.js

```diff
@@ -141,6 +145,39 @@ function wrap (argv, env, workingDir) {
   return unwrap
 }
 
+// handle special cases of shebang, e.g., the
+// 2>/dev/null; exec "`dirname "$0"`/node" "$0" "$@"
+// approach used in Node 5.8.0.
+function fixShebang (options, workingDir) {
+  if (isWindows) return
+  // currently, the only edge-case that we solve
+  // is for Node > v5.8.0.
+  if (/^v0\..*$/.test(process.version)) return
```

Comment:
my thinking is it's worth this risk, to make sure that we apply this hacky fix to as few versions of Node as possible -- I'd rather allow older versions of Node.js continue to have the original behavior.

What do you think?